src/index.js

```diff
@@ -111,9 +114,11 @@ const ELEMENT_IDS = {
     modal: "modal",
     modalControlWrapper: "modal-control-wrapper",
     phoneNumber: "phone-number",
+    recents: "recents",
```

Comment:
tab/view/or however you want to call it, I would make them stand out from all other ids, for example by adding some prefix or suffix dedicated only to such type of elements, for example:

`recentsTab:' recents-tab-wrapper`